### PR TITLE
 [6.2.z] additional katello-backup tests

### PR DESCRIPTION
**Pull request by pondrejk**
Related to issue #4456 
Unfortunately, just one of the new tests is not blocked by https://bugzilla.redhat.com/show_bug.cgi?id=1384901 (that one passes 🙂 ). Anyway, putting do_not_merge, until the next snap that will hopefully let me show some more results.

### CODE REVIEW

**Comment by pondrejk** [comment]
Ready for review, test results on 6.2.9-1 (standalone job 412):

```
tests/foreman/sys/test_hot_backup.py::HotBackupTestCase::test_positive_incremental PASSED

tests/foreman/sys/test_hot_backup.py::HotBackupTestCase::test_positive_incremental_skip_pulp PASSED

tests/foreman/sys/test_hot_backup.py::HotBackupTestCase::test_positive_online_backup_with_directory_created PASSED

tests/foreman/sys/test_hot_backup.py::HotBackupTestCase::test_positive_online_backup_with_existing_directory PASSED

tests/foreman/sys/test_hot_backup.py::HotBackupTestCase::test_positive_online_incremental <- robottelo/decorators/__init__.py SKIPPED

tests/foreman/sys/test_hot_backup.py::HotBackupTestCase::test_positive_online_incremental_skip_pulp PASSED

tests/foreman/sys/test_hot_backup.py::HotBackupTestCase::test_positive_online_skip_pulp PASSED

tests/foreman/sys/test_hot_backup.py::HotBackupTestCase::test_positive_skip_pulp PASSED

 generated xml file: /home/jenkins/workspace/satellite6-standalone-automation/foreman-results.xml 
=================== 5 tests deselected by "-m 'not stubbed'" ===================
============= 7 passed, 1 skipped, 5 deselected in 432.22 seconds ==============
```

Apart from stubs, there is test_positive_online_incremental skipped due to https://bugzilla.redhat.com/show_bug.cgi?id=1435333

File: robottelo/helpers.py [hunk 1]
```diff
@@ -413,3 +414,37 @@ def default_url_on_new_port(oldport, newport):
...
+        'pulp_celerybeat',
+        'pulp_resource_manager',
+        'pulp_streamer',
+        'pulp_workers',
+        'qdrouterd',
+        'qpidd',
+        'smart_proxy_dynflow_core',
+        'squid',
+        'tomcat6' if major_version == RHEL_6_MAJOR_VERSION else 'tomcat',
+    )
```
Comment (ldjebran) [inline robottelo/helpers.py]: perhaps this service should be gotten from a function like get_host_services or get_sat_services as they are used in more than one place

Comment (pondrejk) [inline robottelo/helpers.py]: Thank you @ldjebran, I forgot this is also being used in test_installer.py. After some checking I'd like to leave this PR as is and change test_installer to use this helper function in a separate PR.

**Comment by rplevka** [comment]
ACK
@ldjebran are you happy with this? If so, please merge

**Pull request merged**